code/src/components/AlbumCard.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import ArtistName from './ArtistName';
+
+import { ReactComponent as Heart} from "../icons/heart.svg";
+import { ReactComponent as Play} from "../icons/play.svg";
+import { ReactComponent as Dots} from "../icons/dots.svg";
+
+// This function will be called after each artist name display and will determine what kind of seperator should be used
+const insertSeperator = (index, length) => {
+    if (index === length - 2) { //Between the second to last and last artist " & " will show
+        return " & ";
+      } else if (index < length - 2) { //If more than 2 artists, they will seperate with ","
+        return ", ";
+      }
+}
+
+const AlbumCard = (props)=> {
+    return (
+        <div className="album-card-container">
+            <div className="album-image-container">
+                <img className="album-image" src={props.image} alt="Album cover" />
+                <div className="album-image-overlay"> {/* In order to style svg-icons we imported them as react-components*/}
```

Comment:
Would be there not another solution to style the icons? f.e.x by defining a class for each one